src/atoms/Size/__tests__/Size.web.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import View from '../../View';
+import Size from '../index';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
```

Comment:
the test that actually takes scales as input is missing for example
`height={1}` will go through `makePxValue` and become `8px`.
So you need to write a test that actually takes these numeric values apart from `'10px'`